app/models/admin.rb

```diff
@@ -1,7 +1,7 @@
 class Admin < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  # :registerable
+  # :registerable, :recoverable,
   devise :database_authenticatable,
-         :recoverable, :rememberable, :validatable
+         :rememberable, :validatable
```

Comment:
I think that this is not needed and it is safer to recover the admin account by changing the password using a console, then giving phishers the ability to recover via email :)